includes/class-wc-payments-token-service.php

```diff
@@ -77,7 +77,7 @@ public function add_token_to_user( $payment_method, $user ) {
 	 */
 	public function add_payment_method_to_user( $payment_method_id, $user ) {
 		$payment_method_object = $this->payments_api_client->get_payment_method( $payment_method_id );
-		return $this->add_token_to_user( $payment_method_object, wp_get_current_user() );
+		return $this->add_token_to_user( $payment_method_object, $user );
```

Comment:
All around this file there were a lot of calls to `wp_get_current_user()`. That obviously will break when the current user is not the same person as the owner of the token. @luizreis Do these changes look good to you? I assume it was a simple oversight.

Reply:
Yes, they do! Thanks a bunch for catching and fixing that! 💯